### PR TITLE
Nextcloud runs only on PHP 5.6+

### PR DESCRIPTION
**Pull request by MorrisJobke**
cc @LukasReschke @rullzer @nickvergessen

### CODE REVIEW

File: console.php [hunk 1]
```diff
@@ -31,10 +31,10 @@
...
 define('OC_CONSOLE', 1);
 
-// Show warning if a PHP version below 5.4.0 is used, this has to happen here
-// because base.php will already use 5.4 syntax.
-if (version_compare(PHP_VERSION, '5.4.0') === -1) {
-	echo 'This version of ownCloud requires at least PHP 5.4.0'.PHP_EOL;
+// Show warning if a PHP version below 5.6.0 is used, this has to happen here
+// because base.php will already use 5.6 syntax.
+if (version_compare(PHP_VERSION, '5.6.0') === -1) {
+	echo 'This version of Nextcloud requires at least PHP 5.4.0'.PHP_EOL;
```
Comment (rullzer) [inline console.php]: don't lie ;) Its 5.6

Comment (MorrisJobke) [inline console.php]: 🙈 Fixed

**Comment by nickvergessen** [comment]
👍

**Comment by rullzer** [comment]
LGTM

**Pull request merged**